app/src/seed_script.py

```diff
@@ -0,0 +1,195 @@
+                                augue gravida. Etiam ac pretium urna. Pellentesque posuere hendrerit turpis sed suscipit. Suspendisse 
+                                ultrices, augue in tristique ultricies, ligula diam facilisis justo, eget consequat mauris ipsum sit 
+                                amet nunc. Etiam non nisl justo. Donec dignissim ullamcorper maximus. Fusce rhoncus volutpat augue.  
+                            </p>
+                            """,
+            textbook_name="sample.pdf",
+            textbook_pages="<li>Chapter 3 - Read Pages 44 to 66</li><li>Complete Exercises 1-10</li>",
+            practice_content="Spaced repetition learning here, maybe through some mini interactable question module here (for no marks, just practice)?",
+            topic_id=module1_topic2.id
+        )
+        db.session.add(module1_topic2_lesson)
+        db.session.commit()
+
+        module1_topic3 = Topic(name="Prime Factorization", module_id=module1.id)
+        db.session.add(module1_topic3)
+        db.session.commit()
+
+        module1_topic3_lesson = Lesson(
+            title="Factors & Multiples",
+            learning_objective="<li>Understand and Identify Factors</li><li>Apply factorization in real world contexts</li><li>Utilize factors to Simplify Fractions</li><li>Explore and apply prime factorization</li>",
+            video_filename="video.mp4",
+            thumbnail_filename="thumbnail.png",
+            textbook_name="sample.pdf",
+            textbook_pages="<li>Chapter 3 - Read Pages 44 to 66</li><li>Complete Exercises 1-10</li>",
+            practice_content="Spaced repetition learning here, maybe through some mini interactable question module here (for no marks, just practice)?",
+            topic_id=module1_topic3.id
+        )
+        db.session.add(module1_topic3_lesson)
+        db.session.commit()
+
+        create_file_paths(course.id, module1.id, module1_topic1.id)
+
+        module2 = Module(name="Patterns", course_id=course.id)
+        db.session.add(module2)
+        db.session.commit()
+
+        module3 = Module(name="Ratios and Rates", course_id=course.id)
+        db.session.add(module3)
+        db.session.commit()
+
+        module4 = Module(name="Percentages", course_id=course.id)
+        db.session.add(module4)
+        db.session.commit()
+
+        module5 = Module(name="Exponents Intro & Order of Operations", course_id=course.id)
+        db.session.add(module5)
+        db.session.commit()
+
+        module6 = Module(name="Variables & Expressions", course_id=course.id)
+        db.session.add(module6)
+        db.session.commit()
+
+        module7 = Module(name="Equations & Inequalities", course_id=course.id)
+        db.session.add(module7)
+        db.session.commit()
+
+        module8 = Module(name="Proportional Relationships", course_id=course.id)
+        db.session.add(module8)
+        db.session.commit()
```

Comment:
I'd get rid of these, having empty Module tabs gives off an unpolished feel/experience for the user